opensarlab_lib/aoi_selectors.py

```diff
@@ -1,16 +1,26 @@
 from typing import List, Optional, Union, Tuple
 
+import cartopy.crs
+import cartopy.feature as cfeature
+from cartopy.io.img_tiles import GoogleTiles, OSM
 import matplotlib
 import matplotlib.pyplot as plt
 from matplotlib.widgets import RectangleSelector
 import matplotlib.patches as patches
-
-import cartopy.crs
-import cartopy.feature as cfeature
 import numpy as np
+import pyproj
 
 plt.rcParams.update({'font.size': 12})
 
+class EsriImagery(GoogleTiles):
+    def _image_url(self, tile):
+        x, y, z = tile
+        url = (
+            'https://server.arcgisonline.com/ArcGIS/rest/services/'
+            'World_Imagery/MapServer/tile/{z}/{y}/{x}'
+        ).format(z=z, y=y, x=x)
```

Comment:
Does the library need us to use `.format()` here instead of an f-string?

Reply:
Good catch!